AlfrescoApp/Model/Managers/RealmSyncManager+Refresh.m

```diff
@@ -167,6 +165,15 @@ - (void)updateDataBaseForChildNode:(AlfrescoNode *)childNode withParent:(Alfresc
     [realm commitWriteTransaction];
 }
 
+- (void)markAsNotTopLevelSyncNodeInfoWithIdentifier:(NSString *)nodeIdentifier
```

Comment:
Perhaps a better name for this method would be: removeTopLevelNodeFlagFomNodeWithIdentifier: